src/routes/wallet/wallet.routes.js

```diff
@@ -217,6 +218,11 @@ walletRouter.post(
     'outputs.*.data': {
       in: ['body'],
       isString: true,
+      isLength: {
+        options: {
+          max: MAX_DATA_SCRIPT_LENGTH
+        }
```

Comment:
```suggestion
        options: {
          max: MAX_DATA_SCRIPT_LENGTH,
        },
```

Reply:
Since our linter rules are not currently explicit about this situation, we've decided to move on with this PR and refactor the lint rules (and the files) in another PR